packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1088,6 +1088,18 @@ base class PipelineOwner with DiagnosticableTreeMixin {
   bool _shouldMergeDirtyNodes = false;
   List<RenderObject> _nodesNeedingLayout = <RenderObject>[];
 
+  /// The [RenderObject]s which need to be laid out in the next [flushLayout] pass.
+  ///
+  /// [RenderObject]s with [RenderObject.isRepaintBoundary] are added
```

Comment:
This doesn't sound right. For layout we collect relayout boundaries instead of repaint boundaries.

Reply:
Woops! Fixed